src/current/v24.3/ldap.md

```diff
@@ -0,0 +1,206 @@
+
+Q: Should we specify any LDAP-specific recommendations/considerations for the first four fields (host, all, all, all) or enhance the authentication page to better cover them?
+
+#### Step 3: Configure TLS (Optional)
+
+If using LDAPS, configure the necessary certificates.
+
+Set a custom CA certificate if needed:
+
+~~~ sql
+    SET CLUSTER SETTING server.ldap_authentication.domain_ca = '<PEM_ENCODED_CA_CERT>';
+~~~
+
+Configure a client certificate for mTLS if required:
+
+~~~ sql
+    SET CLUSTER SETTING server.ldap_authentication.client.tls_certificate = '<PEM_ENCODED_CERT>';
+    SET CLUSTER SETTING server.ldap_authentication.client.tls_key = '<PEM_ENCODED_KEY>';
+~~~
+
+#### Step 4: Create Database Users
+
+Q: Confirm: Does a new customer need to create one user here or all users in bulk? 
+
+Create a CockroachDB user that matches the LDAP username:
+
+~~~ sql
+CREATE ROLE username LOGIN;
+~~~
+
+### Connecting with LDAP Authentication
+
+Users can connect using their LDAP credentials:
+
+~~~ bash
+# Method 1: Password in connection string
+cockroach sql --url "postgresql://username:ldap_password@host:26257" --certs-dir=certs
+
+# Method 2: Password in environment variable
+export PGPASSWORD='ldap_password'
+cockroach sql --url "postgresql://username@host:26257" --certs-dir=certs
+~~~
+
+## LDAP Authorization
+
+### Overview
+
+LDAP authorization allows CockroachDB to use LDAP/AD groups as the source of truth for [role](https://www.cockroachlabs.com/docs/stable/security-reference/authorization#roles) assignments. When enabled:
+
+1. During login, CockroachDB fetches the user's group memberships from LDAP.
+2. Groups are mapped to CockroachDB roles using the group's Common Name (CN).
+3. The user is automatically granted membership to corresponding roles.
+4. Any existing role memberships not matched to LDAP groups are revoked.
+
+### Configuration
+
+#### Step 1: Enable LDAP Authorization
+
+Add the `ldapgrouplistfilter` parameter to your HBA configuration:
+
+~~~ sql
+SET CLUSTER SETTING server.host_based_authentication.configuration = '
+host    all    all    all    ldap    ldapserver=ldap.example.com 
+    ldapport=636 
+    "ldapbasedn=ou=users,dc=example,dc=com" 
+    "ldapbinddn=cn=readonly,dc=example,dc=com" 
+    ldapbindpasswd=readonly_password 
+    ldapsearchattribute=uid 
+    "ldapsearchfilter=(memberof=cn=cockroachdb_users,ou=groups,dc=example,dc=com)"
+    "ldapgrouplistfilter=(objectClass=groupOfNames)"
+host    all    root    0.0.0.0/0    password';
+~~~
+
+The `ldapgrouplistfilter` parameter defines which LDAP objects should be considered as groups.
+
+Q: Get more specific/realistic example from Eng?
+
+#### Step 2: Create Matching Roles
+
+Create CockroachDB roles that match your LDAP group names and grant appropriate privileges to each role.
+
+For an LDAP group `cn=db_admins,ou=groups,dc=example,dc=com`:
+
+~~~ sql
+CREATE ROLE db_admins;
```

Comment:
Another caveat is the `CN` field of the distinguished name needs to be sql username compatible which is provided here: https://www.cockroachlabs.com/docs/v24.2/create-user#user-names. 
cc: @pritesh-lahoti